README.md

```diff
@@ -19,11 +19,22 @@ If you need help using or installing Node.js, please use the
 
 ## Resources for Newcomers
 
+### Official Resources
+
 * [Website](https://nodejs.org/en/)
 * [Contributing to the project](./CONTRIBUTING.md)
-* IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4)
 * IRC (node core development): [#node-dev on chat.freenode.net](https://webchat.freenode.net?channels=node-dev&uio=d4)
 
+### Unofficial Resources
+
+* IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4). Please
+see http://nodeirc.info/ for more information regarding the `#node.js` IRC channel.
+
+*Please note that unofficial resources are neither managed by -- nor necessarily
```

Comment:
It's simply a lazy em dash, e.g. —

On Jul 20, 2016 2:33 PM, "Сковорода Никита Андреевич" <
notifications@github.com> wrote:

> In README.md
> <https://github.com/nodejs/node/pull/7810#discussion_r71611032>:
>
> >  * IRC (node core development): [#node-dev on chat.freenode.net](https://webchat.freenode.net?channels=node-dev&uio=d4)
> >
> > +### Unofficial Resources
> > +
> > +* IRC (general questions): [#node.js on chat.freenode.net](https://webchat.freenode.net?channels=node.js&uio=d4). Please
> > +see http://nodeirc.info/ for more information regarding the `#node.js` IRC channel.
> > +
> > +*Please note that unofficial resources are neither managed by -- nor necessarily
>
> What do two minus signs mean? Is that commonly used or is there an UTF
> char for that?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/7810/files/4c05ed56a3dcb5a0b8e41c1edaaa6f0b32df0d9d#r71611032>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAa2ebA1-m1dsn2bl8VTY9gdBvIhX4_nks5qXpQ-gaJpZM4JQ5v8>
> .
>